pages/restmail.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import time
+import requests
+
+
+def get_login_link(username, message_count=1, timeout=60):
```

Comment:
I'm okay with this, but I'd prefer `restmail.py` to be specific to restmail.net and not include extraction of the login link. It would be better to just return the mail, and do the extraction in a test fixture.